### ISSUE TITLE
A11y_MSRCWeb_LandingPage_ScreenReader: Sorting functionality is not announced for the column headers  "Short Description","Submission Status","Date Submitted","Sumission ID".............

### ISSUE THREAD

**Issue by nirmalsahoo** [OfficeDev/office-ui-fabric-react#10404]
“[Check out Accessibility Insights! ](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
Application Name: MSRC Web
Windows Version: Windows10
Versions : Microsoft Edge 42.17134.1.0   
 Application URL:  [https://msrc-ppe.microsoft.com](https://msrc-ppe.microsoft.com)

Steps to Reproduce:
1.Start ScreenReader.
2.Launch "MSRC Web" Web Application or directly hit URL- https://msrc-ppe.microsoft.com/create-report
3.Go to the HomePage i.e. Landing Page 
4.Go to the table-->data under the column headers "Date Submitted"-->Press Enter to sort.
5.Verify that all the actions are announced by the screen reader.
 

**Actual:**
Sorting functionality is not announced for the column header "Short Description","Submission Status","Date Submitted"....
 
Though sorting functionality is implemented on the column header of the table "Short Description","Submission Status","Date Submitted".... still no information is read out by the screen reader regarding the same.
 
When the column header is hit using "Enter", the sign changes from ascending to descending and vice versa but nothing regarding the same is read by the screen reader
 
**Expected:**
Sorting functionality should be announced for the column header "Short Description","Submission Status","Date Submitted"....
 
Since sorting functionality is implemented on the column header of the table "Short Description","Submission Status","Date Submitted".... therefore information should be read out by the screen reader regarding the same.
 
When the column header is hit using "Enter", the sign changes from ascending to descending and vice versa therefore regarding the same should be read out by the screen reader.
 
**User Impact:**
People who are blind and people who have low vision may not be able to understand information if it is conveyed by shape and/or location. Providing additional information other than shape and/or location will allow them to understand the information conveyed by shape and/or alone. 
 
**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx)

MAS Reference:
MAS1.3.1-
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4})

Video Link:
[A11y_MSRCWeb_LandingPage_ScreenReader.pptx](https://github.com/OfficeDev/office-ui-fabric-react/files/3595350/A11y_MSRCWeb_LandingPage_ScreenReader.pptx)

**Comment by natalieethell** [comment]
Hey @nirmalsahoo thank you for opening this issue! This is definitely huge missing functionality that we want DetailsList to have. There's actually an existing thread on another open issue for the same problem: https://github.com/OfficeDev/office-ui-fabric-react/issues/7786.  

To sum up the thread, DetailsHeader currently has `aria-sort` attributes on column headers that can be sorted, but screen readers do not seem to pick up on them. What we decided would probably be our solution is integrating the Announced component to fill in this accessibility gap. Now that we have the Announced component ready, I think it's time to make those changes.

I'll close this issue out in favor of https://github.com/OfficeDev/office-ui-fabric-react/issues/7786 so that we can keep everything in one place, but thank you again for bringing this back to our attention!
